src/components/cart/cart.tsx

```diff
@@ -37,11 +47,17 @@ export const Cart = () => {
                         ))}
                     </div>
                     <div className={styles.subtotalCheckout}>
+                        {checkoutAttempted && someItemsOutOfStock && (
+                            <div className={styles.errorMessage}>Some items are out of stock</div>
+                        )}
                         <label className={styles.subtotalLabel}>
                             <span>Subtotal:</span>
                             {cartTotals?.priceSummary?.subtotal?.formattedConvertedAmount}
                         </label>
-                        <button className={Classnames(commonStyles.primaryButton, styles.checkout)} onClick={checkout}>
+                        <button
+                            className={classnames(commonStyles.primaryButton, styles.checkoutButton)}
```

Comment:
this change will have a conflict with this:
https://github.com/codux-templates/e-commerce-remix/pull/61

we have to not forget to fix it properly after merging